server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -441,16 +441,21 @@ public static VolumeManager get(AccumuloConfiguration conf, final Configuration
     return new VolumeManagerImpl(volumes, conf, hadoopConf);
   }
 
+  @SuppressWarnings("deprecation")
+  private static boolean inSafeMode(DistributedFileSystem dfs) throws IOException {
+    // Returns true when safemode is on; this version of setSafeMode was deprecated in Hadoop 3.3.6,
+    // because SafeModeAction enum was moved to a new package, and this deprecated method was
+    // overloaded with a version of the method that accepts the new enum. However, we can't use that
+    // replacement method if we want to continue working with versions less than 3.3.6, so we just
+    // suppress the deprecation warning.
```

Comment:
There is another change in HadoopLogCloser referenced in #3494, that did not require a similar @SuppressWarnings change?

Reply:
The build passed without any other warnings. So I guess any APIs related to what is proposed in #3494 that we're currently using have not been deprecated (or they are already suppressed). #3494 isn't implemented yet, though. When it is, it may require some other change, in addition to being able to drop this suppression and use the overloaded new version.